Makefile

```diff
@@ -630,11 +630,11 @@ generate-go-conversions: $(CONVERSION_GEN) ## Generate conversions go code
 		--output-file-base=zz_generated.conversion \
 		--go-header-file=./hack/boilerplate.go.txt
 
-.PHONY: generate-package-config ## Generate the default package config CR e.g. make generate-package-config apiGroup=cni.tanzu.vmware.com kind=AntreaConfig version=v1alpha1 tkr=v1.23.3---vmware.1-tkg.1 namespace=tkg-system
+.PHONY: generate-package-config ## Generate the default package config CR e.g. make generate-package-config apiGroup=cni.tanzu.vmware.com kind=AntreaConfig version=v1alpha1 tkr=v1.23.3---vmware.1-tkg.1 namespace=tkg-system iaas=vsphere
```

Comment:
suggestion: would it be possible to add some comments about `iaas`. So that whoever tries to use this make target knows what values are supported and when to `vsphere` or the other. Thanks